unicodetools/src/main/java/org/unicode/text/UCD/TestUnicodeInvariants.java

```diff
@@ -46,6 +46,9 @@ public class TestUnicodeInvariants {
     private static int showRangeLimit = 20;
```

Comment:
Nice addition, and cleanup

Reply:
Note that I split that out into its own PR #646, which will thus need its own approval.